### ISSUE TITLE
Use new :crypto API

### ISSUE THREAD

**Issue by wuhkuh** [dwyl/phoenix-ecto-encryption-example#22]
According to the [:crypto documentation](http://erlang.org/doc/apps/crypto/new_api.html) `block_encrypt` and `block_decrypt` are old and should not be used in new applications. We should consider updating the encryption functions to the new API.

**Comment by nelsonic** [comment]
@wuhkuh great suggestion! Thank you for taking time to open this issue.

**Comment by nelsonic** [comment]
"_not deprecated for now, but may be in a future release_":
![image](https://user-images.githubusercontent.com/194400/60563935-a35be380-9d55-11e9-9841-e394a1aaeb35.png)

If anyone has interest/time to update the tutorial, it would be _very_ much appreciated. 👍

**Comment by nelsonic** [comment]
@NikitaAvvakumov kindly created the PR: https://github.com/dwyl/phoenix-ecto-encryption-example/pull/23 to update the `:crypto` syntax in the `README.md` 
we need to update it in the **`code`** 
https://github.com/dwyl/phoenix-ecto-encryption-example/blob/09c0a24972eee3e1262e1129a18cc60e3ea53098/lib/encryption/aes.ex#L33
_and_ bump the Erlang version in `.travis.yml` to `22` so that the build will actually _pass_:
https://github.com/dwyl/phoenix-ecto-encryption-example/blob/09c0a24972eee3e1262e1129a18cc60e3ea53098/.travis.yml#L5

**Comment by plicjo** [comment]
Using Erlang 23+:

```elixir
{ciphertext, tag} = :crypto.crypto_one_time_aead(:aes_gcm, key, iv, to_string(plaintext), @aad, true)
```

gets the tests passing :)

**Comment by nelsonic** [comment]
Sweet. thanks @plicjo ⭐

**Comment by nelsonic** [comment]
Included in #47
